modules/mobile/docs/openapi.yaml

```diff
@@ -1903,13 +1903,16 @@ paths:
   /v0/health/rx/prescriptions/refill:
     put:
       description: Requests refill for prescriptions.
-      parameters:
-        - description: array of prescription ids to request refill
-          example: ["7417954", "6970769", "8398465"]
-          in: query
-          name: ids[]
-          schema:
-            type: array
+      requestBody:
```

Comment:
I wasn't sure how this is supposed to be formatted, so I looked up similar request body docs. They all seem to be formatted like:
```
type: object
required:
  - id
  - emailAddress
properties:
  id:
    type: integer
    example: 42
  emailAddress:
    type: string
    example: "person42@example.com"
```

Is there a reason you didn't break it down into properties?

Reply:
They're probably both correct. Who knows.